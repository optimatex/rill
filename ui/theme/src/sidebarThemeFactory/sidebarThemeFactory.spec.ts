import { getTheme } from 'test';
import { sidebarThemeFactory } from '.';

describe('mainThemeFactory', () => {
  it('should act as the factory', () => {
    // @ts-ignore
    expect(sidebarThemeFactory(getTheme)).toMatchSnapshot();
  });
});
