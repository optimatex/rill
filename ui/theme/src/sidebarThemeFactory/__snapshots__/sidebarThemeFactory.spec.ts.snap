// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mainThemeFactory should act as the factory 1`] = `
Object {
  "column": Object {
    "maxWidth": 150,
  },
  "frame": Object {
    "columnGap": 14,
    "rowGap": 18,
    "sideIndent": 8,
    "verticalIndent": 6,
  },
}
`;
